erizo_controller/test/erizoAgent/erizoAgent.js

```diff
@@ -24,6 +24,7 @@ describe('Erizo Agent', () => {
   };
 
   beforeEach(() => {
+    global.config = { logger: { configFile: true } };
```

Comment:
do you want to enable logs for UTs?

Reply:
No, I just needed `global.config` to be present correctly for the ROV componets, otherwise tests will fail. I noticed that's what we did for erizoController and ErizoJS so I followed that pattern.